src/random.cpp

```diff
@@ -116,7 +116,7 @@ static uint64_t GetRdRand() noexcept
     // RdRand may very rarely fail. Invoke it up to 10 times in a loop to reduce this risk.
 #ifdef __i386__
     uint8_t ok;
-    uint32_t r1, r2;
+    uint32_t r1 = 0, r2 = 0;
```

Comment:
```suggestion
    uint32_t r1 = 0, r2 = 0; // ...
```

 Mabye also add an inline comment to explain why setting to zero is not a bug?

Reply:
Done: https://github.com/bitcoin/bitcoin/compare/23faed094ecf5f77fa250e57464bab2fb089730c..71f183a49b714a28622277fa668d8f9f3dac0aae